addons/message-filters/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Message filtering",
+    "description": "Enables the ability to finely filter messages on the messages page through checkboxes.",
+    "credits": [
+      {
+        "name": "--Explosion--",
+        "link": "https://github.com/explosion-scratch"
+      }
+    ],
+    "userscripts": [
+      {
+        "url": "userscript.js",
+        "matches": ["https://scratch.mit.edu/*"]
+      }
+    ],
+    "tags": [],
+    "enabledByDefault": false
+  }
+  
```

Comment:
Could also be settings for that addon in the settings page as default options, like onion skinning.

Reply:
No, users may want something different easily.

Reply:
Not sure you understood what I was trying to tell you. Keep the current settings in the messaging page, but also create some in the settings page as default options. Not sure if you should, I would ask before doing this.

Reply:
I think it's fine as is, but what I _do_ want to do is add localStorage saving to it.